src/pathfinding_service/model/token_network.py

```diff
@@ -140,6 +141,32 @@ def _calculate_fees(self) -> Optional[List[FeeAmount]]:
 
         return fees
 
+    def _get_matrix_users(self) -> Optional[Dict[str, str]]:
+        # Check node reachabilities
+        user_ids: Dict[str, str] = {}
+        for node in self.nodes:
+            node_user_ids = self.reachability_state.get_userids_for_address(node)
+            checksummed_address = to_checksum_address(node)
+            for user_id in node_user_ids:
+                if self.reachability_state.get_userid_presence(user_id) in [
+                    UserPresence.ONLINE,
+                    UserPresence.UNAVAILABLE,
+                ]:
+                    user_ids[checksummed_address] = user_id
+                    # if a reachable user is found we arbitrarily choose
+                    # this user for the given address. There should not be another user online
+                    break
+
+            if checksummed_address not in user_ids:
```

Comment:
This could else be an `else` to the `for` loop instead of an `if`. But it's fine either way.